client/js/lounge.js

```diff
@@ -321,7 +321,7 @@ $(function() {
 			"whois",
 			"ctcp",
 			"channel_list",
-			"ban_list",
+			"ban_list"
```

Comment:
@xPaw, do we want/need an ESLint rule to prevent those? (i.e. to keep the trailing commas)